Exercises/numbers/numbers.js

```diff
@@ -0,0 +1,50 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+//const decimal = new Number('127');
+
+const binary = '0000000001111111';
+const octal = '0x7F';
+const hexadecimal = '7F'; 
+
+console.log(binary,octal,hexadecimal);
+
+//2. Implement function `extractNumbers` which will leave only numbers in passed string (in the beginning of the string).
+
+function extractNumbers () {
+  price = '999usd';
+  number = parseInt(price); 
+  console.log(number);
+}
+extractNumbers();
+
+//3. Declare a variable with decimal value and convert to binary, octal and hexadecimal.
+
+const numberDecimal = new Number('9');
+
+console.log((numberDecimal).toString(2)); //1001
+console.log((numberDecimal).toString(8)); //011
+console.log((numberDecimal).toString(16)); //9
+
+//4. Use `Math.random` to generate random value from `0` to `100`.
+
+let randomInteger = Math.floor(Math.random() * 101);
+console.log(randomInteger);
+
+//5. Make function which will take a array with numbers as a parameter and calculate average number and round to hundredths.
+
+function arrayCalculation () {
```

Comment:
great! Just run lint here and fix problems

Reply:
and rewrite a bit: `arrayCalculation` should take array as a dynamic parameter instead of hardcoded `values`